examples/getting-started/Vagrantfile

```diff
@@ -8,9 +8,9 @@ Vagrant.require_version ">= 2.0.0"
 # We do not have binary packages for iproute2 with BPF templating support
 # so if we change this version without otherwise providing those packages
 # then this GSG VM will break!
-cilium_version = (ENV['CILIUM_VERSION'] || "v1.4.1")
+cilium_version = (ENV['CILIUM_VERSION'] || "v1.6.6")
```

Comment:
If this works, please delete the outdated comment above this line.

I had previously assumed Cilium was running with systemd in this environment, and that we didn't provide iproute2 here, so thought this would lead to failure to load BPF programs.